iktomi/cli/base.py

```diff
@@ -77,8 +77,6 @@ def perform_auto_complete(commands):
                     k,v = splited
                 elif len(splited) == 1:
```

Comment:
I meant both `--opt`, `--opt=` and `--opt=""` should behave the same way. I case of boolean option it should mean true.